config/metadata/dependencies.yaml

```diff
@@ -6,4 +6,4 @@ dependencies:
 - type: olm.package
   value:
     packageName: ocs-operator
-    version: "4.7.0"
+    version: ">4.7.0 < 4.8.0"
```

Comment:
Not sure, but it might be that the extra space between the "<" and the "4.8.0" might cause problems.

Reply:
Done